examples/prisma-ecommerce/src/App.test.js

```diff
@@ -1,9 +0,0 @@
-import React from 'react';
```

Comment:
Did you remove this file on purpose? If so why?

Reply:
Yes at https://github.com/marcantoine/ra-data-graphql-prisma/pull/12/commits/4ff635ffb060d3cf249cbb4ede142139f3f5615b

It doesn't bring anything, and it's crashing the main test because it's somehow executed when running tests on the main project.

As it doesn't provides any value, there is no point keeping it at all.